scripts/generateNodeReleasesData.mjs

```diff
@@ -0,0 +1,39 @@
+import nodevu from '@nodevu/core';
+import { writeFile } from 'fs';
+
+const getNodeReleasesData = async () => {
+  const nodevuOutput = await nodevu();
+
+  // Filter out those without documented support
+  const majors = Object.values(nodevuOutput).filter(major => {
+    return major?.support;
+  });
+
+  const nodeReleases = majors.map(major => {
+    const latestVersion = Object.values(major.releases)[0];
+
+    return {
+      version: `v${latestVersion.semver.major}`,
+      fullVersion: latestVersion.semver.raw,
+      codename: major.support.codename,
+      initialRelease: major.support.phases.dates.start,
+      ltsStart: major.support.phases.dates.lts,
+      maintenanceStart: major.support.phases.dates.maintenance,
+      endOfLife: major.support.phases.dates.end,
+    };
+  });
+
+  return nodeReleases;
+};
+
+// TODO: Allows running the script from any directory
+writeFile(
```

Comment:
```suggestion
return writeFile(
```

Reply:
This writeFile call should be inside the `getNodeReleasesData` function.

And the function should be called inside `next.data.mjs`.